### ISSUE TITLE
ExUnit deprecation in Elixir 1.5

### ISSUE THREAD

**Issue by blatyo** [victorolinasc/junit-formatter#25]
```
warning: passing GenEvent handlers (JUnitFormatter in this case) in the :formatters option of ExUnit is deprecated, please pass a GenServer instead. Check the documentation for the ExUnit.Formatter module for more information
  (ex_unit) lib/ex_unit/event_manager.ex:34: ExUnit.EventManager.add_handler/3
  (elixir) lib/enum.ex:675: Enum."-each/2-lists^foreach/1-0-"/2
  (elixir) lib/enum.ex:675: Enum.each/2
  (ex_unit) lib/ex_unit/runner.ex:29: ExUnit.Runner.configure/1
  (ex_unit) lib/ex_unit/runner.ex:7: ExUnit.Runner.run/2
```

**Comment by cs-victor-nascimento** [comment]
Ouch... I guess its time to update to latest. I will try to take a look at this during next week.

**Comment by cs-victor-nascimento** [comment]
Being having some difficulties as discussed in #26. Please feel free to tag along there.

**Comment by cs-victor-nascimento** [comment]
Released 2.0.0 with Elixir 1.4+ compatibility which fixes all the warnings. Please, feel free to re-open the issue if there are any other warnings.

**Comment by blatyo** [comment]
Thanks!
